UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -8,6 +8,7 @@
 using Player;
 using Systems.Clothing;
 using UI.CharacterCreator;
+using UnityEditor;
```

Comment:
doesn't look like you need this namespace. If you don't need it, remove it since it will make builds fail because this namespace isn't available for game builds. If you need it, then use a preprocessor to wrap it, like this:

```cs
#IF UNITY_EDITOR
using UnityEditor;
#ENDIF

...

//whatever you are using that namespace for
#IF UNITY_EDITOR
public void MyMethodThatNeedsUnityEditorNamespace()
{
    ...
}
#ENDIF
```

Reply:
Isn't used, gonna remove